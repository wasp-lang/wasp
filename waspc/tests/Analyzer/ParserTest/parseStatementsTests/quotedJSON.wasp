test JSON {=json
  "key": "value"
json=}
