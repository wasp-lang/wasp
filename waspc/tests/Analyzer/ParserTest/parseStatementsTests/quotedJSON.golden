(AST
  (Decl@1:1-3:6 type=test name=JSON
    (Quoter@1:11-3:6 tag=json
      {=
        "key": "value"
      =}
    )
  )
)
